machines/bleShare/scan/scanMachine.ts

```diff
@@ -137,6 +139,26 @@ export const scanMachine =
         inactive: {
           entry: 'removeLoggers',
         },
+        checkStorage: {
+          invoke: {
+            src: () =>
+              Promise.resolve(
```

Comment:
can we extract this code snipped to a new function in the service section

Reply:
Refactored